src/test/java/roomescape/acceptance/ReservationAcceptanceTest.java

```diff
@@ -1,70 +1,156 @@
-    @TestFactory
-    Stream<DynamicTest> pastTimeReservationTest() {
-        LocalDate yesterday = LocalDate.now().minusDays(1);
+    private void initAdminData() {
+        memberRepository.save(AdminFixture.member);
+        reservationTimeRepository.save(AdminFixture.time);
+        themeRepository.save(AdminFixture.theme);
+    }
 
-        return Stream.of(
-                dynamicTest("과거 시간에 대한 예약을 추가한다", () -> postReservation(yesterday.toString(), 1L, 1L, 400))
-        );
+    private void initUserData() {
+        memberRepository.save(UserFixture.member);
+        reservationTimeRepository.save(UserFixture.time);
+        themeRepository.save(UserFixture.theme);
     }
 
-    @DisplayName("이미 예약된 테마와 시간을 예약을 하면, 예외가 발생한다")
-    @Sql("/test-data/reservation.sql")
-    @TestFactory
-    Stream<DynamicTest> duplicateReservationTest() {
-        LocalDate tomorrow = LocalDate.now().plusDays(1);
+    @DisplayName("사용자 권한으로 예약할 수 있다")
+    @Test
+    void when_postReservationWithUser_then_created() {
+        // given
+        String token = tokenProvider.createToken(UserFixture.member);
+
+        Map<?, ?> request = Map.of(
+                "date", UserFixture.tomorrow.toString(),
+                "timeId", 2,
+                "themeId", 2);
+
+        // when
+        Response response = RestAssured.given().log().all()
+                .contentType(ContentType.JSON)
+                .cookie("token", token)
+                .body(request)
+                .when().post("/reservations")
+                .then().log().all()
+                .statusCode(201)
+                .extract().response();
 
-        return Stream.of(
-                dynamicTest("예약을 추가한다", () -> postReservation(tomorrow.toString(), 1L, 1L, 201)),
-                dynamicTest("동일한 예약을 추가한다", () -> postReservation(tomorrow.toString(), 1L, 1L, 400)),
-                dynamicTest("다른 테마를 예약한다", () -> postReservation(tomorrow.toString(), 1L, 2L, 201))
+        // then
+        JsonPath jsonPath = response.jsonPath();
+        assertAll(
+                () -> assertThat(jsonPath.getLong("id")).isNotNull(),
+                () -> assertThat(jsonPath.getString("member.email")).isEqualTo(UserFixture.email),
+                () -> assertThat(jsonPath.getString("date")).isEqualTo(UserFixture.tomorrow.toString()),
+                () -> assertThat(jsonPath.getString("theme.name")).isEqualTo(UserFixture.themeName)
         );
     }
 
-    private long postReservation(String date, Long timeId, Long themeId, int expectedHttpCode) {
-        Map<?, ?> requestBody = Map.of("name", "포비", "date", date, "timeId", timeId, "themeId", themeId);
+    @DisplayName("어드민 권한으로 예약할 수 있다")
+    @Test
+    void when_postReservationWithAdmin_then_created() {
+        // given
+        String token = tokenProvider.createToken(AdminFixture.member);
+        doReturn(Optional.of(token)).when(tokenProvider).extractToken(any());
 
+        Map<?, ?> request = Map.of(
```

Comment:
```suggestion
        Map<String, Object> request = Map.of(
```
타입 명시를 좋아하시지 않는건가요 😂

Reply:
피케이는 `?`를 좋아하시지 않은 건가요 😂 피케이께서 지양하시는 이유가 궁금해요! 

이 부분도 의도하고 작성한 코드에요. 테스트 코드에서 값 자체가 중요하지 타입은 아니라고 생각해요. 지금은 해당되지 않지만 만약 명시해야 하는 타입이 긴 경우에 타입을 명시하면 가독성이 낮아진다는 단점이 있어요. 현재 테스트 코드에서 집중하고 싶은 부분은 타입이 아니라고 생각해서 와일드 카드를 사용했어요!

Reply:
지양하는 이유는 앞선 코멘트에 남겼습니다!
망쵸가 말해주신 것처럼 이 부분은 타입이 그리 중요하진 않아요.
저는 그럼에도 이런건 팀 컨벤션으로 `제네릭과 와일드카드는 필요할 때 사용하자` 라는 컨벤션을 둘 것 같네요.
타입이 눈에 보여야 `cmd+b` 로 구현 부분을 보기도 쉽거든요.
정답이 있는 문제는 아니니 꼭 수정하실 필요는 없습니다!